tests/conftest.py

```diff
@@ -10,6 +10,11 @@
 from .basic_env import TestDirFixture
 
 
+# Prevent updater and analytics from running their processes
+# NOTE: manually converting to str for Windows/Python 2
+os.environ[str("DVC_TEST")] = str("true")
```

Comment:
Please use `cast_bytes_py2` like so:

```
from dvc.utils.compat import cast_bytes_py2

os.environ[cast_bytes_py2("DVC_TEST")] = cast_bytes_py2("true")
```

and you won't even need that note :slightly_smiling_face: